src/style/design-tokens/theme-compatibility.util.spec.js

```diff
@@ -0,0 +1,36 @@
+import themeCompatibility from "./theme-compatibility.util";
+import mintTheme from "../themes/mint";
+
+describe("themeCompatibility", () => {
+  it("should create clone given theme before extending it", () => {
+    const output = themeCompatibility(mintTheme);
+
+    expect(output).not.toBe(mintTheme);
+  });
+
+  it("should extend object that", () => {
+    const output = themeCompatibility(mintTheme);
+
+    expect(output.colors).toBe(mintTheme.colors);
```

Comment:
We can probably just move these 3 assertions to the first `it` block, or change the description of this block to something more meaningful

Reply:
oh, sure. Description is terrible.

Reply:
Unresolving, both these test descriptions could be improved.